app/templates/components/popular-tags.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="col-md-3">
+  <div class="sidebar">
```

Comment:
i would think the layout elements should be outside the component. will make it easier to re-use this if we have to render tags elsewhere.